lib/govuk_publishing_components.rb

```diff
@@ -20,6 +20,7 @@
 require "govuk_publishing_components/presenters/step_by_step_nav_helper"
 require "govuk_publishing_components/presenters/page_with_step_by_step_navigation"
 require "govuk_publishing_components/presenters/public_layout_helper"
+require "govuk_publishing_components/presenters/layout_footer_helper"
```

Comment:
Nitpick: could this be positioned alphabetically? I know not all of the others are, but they're sort of mostly alphabetical...